styles.css

```diff
@@ -0,0 +1,34 @@
+* {
+  font-family: "Raleway";
+}
+
+body {
+  background-image: url(https://images.unsplash.com/photo-1526289034009-0240ddb68ce3?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=871&q=80);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+}
+
+#title {
+  font-weight: bold;
+  margin-top: 50px;
+  margin-bottom: 10px;
+}
+
+#description {
+    font-style: italic;
+}
+
+#title,
+#description {
+  color: white;
+}
+
+.form {
+  background-color: rgb(24 44 52 / 81%);
+  color: rgb(255, 255, 255);
+  border-radius: 5px;
+  margin-top: 50px;
+  margin-bottom: 50px;
+}
```

Comment:
- [ ] add an extra space at the end of the CSS block

Reply:
- [ ] It is best practice to always leave an extra space after the last CSS block. Why don't you add an extra space to your CSS block below the `.form` class?